packages/service-clients/end-to-end-tests/azure-client/src/test/ephemeralSummaryTrees.ts

```diff
@@ -0,0 +1,3499 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Summary tree for the following test case:
+ * Fluid Audience: Can find original member
+ */
+export const findOriginalMember = {
```

Comment:
A lot of these are exactly the same, or close enough. For tests with the same schema and no pre-attach changes, we can set them all to the same value.

I like exporting all the different test names, but redefining the same tree feels unnecessary.

Reply:
Updated it now to only define five trees, then assign those trees to the different exported test names